kafka_utils/kafka_rolling_restart/main.py

```diff
@@ -70,6 +70,14 @@ def parse_opts():
         '-c',
         help='cluster name, e.g. "uswest1-devc" (defaults to local cluster)',
     )
+    parser.add_argument(
+        '--broker-ids',
+        '-b',
+        required=False,
+        type=int,
+        nargs='+',
+        help='space separated broker IDs to restart (optional, will restart all Kafka brokers in cluster if not specified)',
```

Comment:
Looks great. Minor comment for doc change.
I think it will restart all brokers based on whether -n is provided or not.